packages/arb-token-bridge-ui/src/hooks/useArbStats.ts

```diff
@@ -0,0 +1,32 @@
+import useSWR from 'swr'
+import { getAPIBaseUrl } from '../util'
+import { ArbStats } from '../pages/api/arbstats'
+import { useNetworksAndSigners } from './useNetworksAndSigners'
+
+export const useArbStats = () => {
+  const {
+    l2: {
+      network: { chainID: l2ChainId }
+    }
+  } = useNetworksAndSigners()
+
+  return useSWR(
+    'arbstats',
+    async () => {
+      const response = await fetch(
+        `${getAPIBaseUrl()}/api/arbstats?l2ChainId=${l2ChainId}`,
```

Comment:
Updated. `getAPIBaseUrl` doesn't really need to be a part of the key because it's environment specific (test / prod) and that will never change while the webapp is running.